src/amo/pages/Guides/index.js

```diff
@@ -251,7 +254,15 @@ export const getSections = ({
   }
 };
 
-export const getContent = (slug: string, i18n: I18nType): GuideType | null => {
+export const getContent = ({
+  slug,
+  i18n,
+  sections,
+}: {|
+  slug: string,
+  i18n: I18nType,
+  sections?: Array<SectionsType>,
```

Comment:
Why not keeping `_sections` everywhere since it is mostly for testing purpose?

Reply:
well  `getGuidesSections`'s sections is not used for testing. are you suggesting i rename that too? or sections just within `getContent`?

Reply:
ah ok, sorry I got confused.